Assets/Scripts/ssjAug2018/DeilveryTarget/DeliveryTargetSpawnPoint.cs

```diff
@@ -0,0 +1,36 @@
+﻿using pdxpartyparrot.Core.Actors;
```

Comment:
No need for this class, we don't be spawning the targets. They'll be pre-placed and static in the world